lib/Traits/CkanCatalogGroupTraits.ts

```diff
@@ -0,0 +1,66 @@
+import CkanSharedTraits from "./CkanSharedTraits";
+import CatalogMemberTraits from "./CatalogMemberTraits";
+import GroupTraits from "./GroupTraits";
+import ModelTraits from "./ModelTraits";
+import mixTraits from "./mixTraits";
+import UrlTraits from "./UrlTraits";
+import primitiveTrait from "./primitiveTrait";
+import primitiveArrayTrait from "./primitiveArrayTrait";
+import objectTrait from "./objectTrait";
+import anyTrait from "./anyTrait";
+import modelReferenceArrayTrait from "./modelReferenceArrayTrait";
+import JsonObject from "../Core/Json";
```

Comment:
This import is wrong, and is renaming `JsonValue` to `JsonObject` locally. I've changed this to:

```ts
import { JsonObject } from "../Core/Json";
```

and now I can remove the `// @ts-ignore`